README.md

```diff
@@ -450,7 +450,7 @@ based on evidence that it will make money in the future.
     At the end of the three rounds, there are more outstanding shares-roughly 70% more in this
     case, since 120%×120%×120% is 173%-and each shareholder owns proportionally less of the
     company.
-- 🄳 The [**valuation**](https://en.wikipedia.org/wiki/Valuation_(finance)) of the company is the
+- 🄳 The [**valuation**](https://en.wikipedia.org/wiki/Valuation_(finance) of the company is the
```

Comment:
This is a common issue — there need to be two parens at the end, one for theURL and one for the close link.

Reply:
Yah, I had a bunch of (wrong) double parens because atom added them though my pasted urls already had them. Guess I was a little aggressive on clean up! Thanks!